hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStore.java

```diff
@@ -217,8 +217,7 @@ public void testLoopInLinkBuckets() throws Exception {
   private void createLinkBucket(OzoneVolume sourceVolume, String sourceBucket,
                                 String linkBucket) throws IOException {
     BucketArgs.Builder builder = BucketArgs.newBuilder();
-    builder.setBucketLayout(BucketLayout.DEFAULT)
```

Comment:
ah BucketLayout.DEFAULT = LEGACY.  Then it's tricky. Does it mean the bucket layout is defined inconsistently?